pom.xml

```diff
@@ -60,7 +60,7 @@
           -   version.org.wildfly.core.wildfly-controller-client
           -->
 
-        <fabric8.generator.from>registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift</fabric8.generator.from>
+        <jkube.generator.from>registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift:1.8</jkube.generator.from>
```

Comment:
I wouldn't add the image tag here. We always want the latest released image I'd say.